examples/Backtesting/botorch_analytical.py

```diff
@@ -33,8 +38,8 @@
 # For the full simulation, we need to define some additional parameters.
 # These are the number of Monte Carlo runs and the number of experiments to be conducted per run.
 
-N_MC_ITERATIONS = 2
-N_DOE_ITERATIONS = 2
+N_MC_ITERATIONS = 2 if "SMOKE_TEST" in os.environ else 12
```

Comment:
can you do a `SMOKE_TEST = "SMOKE_TEST" in os.environ` upfront to avoid repeating the os lookups?

Reply:
Will do.